llama_index/agent/react/base.py

```diff
@@ -70,12 +73,13 @@ def from_tools(
     ) -> "ReActAgent":
         tools = tools or []
         chat_history = chat_history or []
-        llm = llm or OpenAI(model="gpt-3.5-turbo-0613")
+        memory = memory or memory_cls.from_defaults(chat_history)
```

Comment:
Nit: should probably use the kwarg here

Reply:
just noticed that we are also not passing llm as kwarg into the `from_defaults`, which means that we are not properly setting the max size of the memory according to the LLM.

Slipping in this fix for other places where we instantiate a memory object as well.

Reply:
Good catch!